compliance_checker/tests/test_imos.py

```diff
@@ -680,6 +680,14 @@ def test_check_data_variables(self):
         for i in range(1,5):
             self.assertFalse(ret_val[i].value)
 
+    def test_check_quality_control_variable_matches_variable(self):
+        self.imos.setup(self.test_variable_dataset)
+        ret_val = self.imos.check_quality_control_variable_matches_variable(self.test_variable_dataset)
+        self.assertTrue(ret_val != None)
+        self.assertTrue(len(ret_val) == 4)
```

Comment:
If one of these fails, you'll get more helpful output if you use more specific assert statements (see [unittest documentation](https://docs.python.org/2/library/unittest.html#test-cases):
``` python
    self.assertIsNotNone(ret_val)
    self.assertEqual(len(ret_val), 4)
```